src/govuk/tools/rebrand.test.js

```diff
@@ -0,0 +1,136 @@
+const { compileSassString } = require('govuk-frontend-helpers/tests')
```

Comment:
The tests will need looking at as this is expecting the repo to be structured as per v5, with a separate govuk-frontend-helpers package.